src/components/RepoSubmission.tsx

```diff
@@ -12,6 +13,7 @@ export declare interface RepoSubmissionProps {
 
 const RepoSubmission = ({ isFormOpen, handleFormOpen }: RepoSubmissionProps): JSX.Element => {
   const { user } = useSupabaseAuth();
```

Comment:
this is `currentUser.user`, the hook could be manipulated to directly export `currentUser.access_token` and deduplicate some code below

Reply:
Just push an update to export token from the hook.